semantic/src/site/elements/loader.overrides

```diff
@@ -1,3 +1,13 @@
 /*******************************
          Site Overrides
 *******************************/
+
+.ui.dimmer .ui.loader {
+    color: @loaderTextColor;
+  }
+.ui.dimmer .ui.loader:before {
+    border: @loaderLineWidth solid @loaderFillColor;
+}
+.ui.dimmer .ui.loader:after {
+    border-color: @shapeBorderColor;
+}
```

Comment:
I've no idea why the default Semantic Loaders had different colour scheme when launched within a dimmer (Modal), but this just resets it back to the same as the standard Loader.